trestlebot/transformers/csv_to_yaml.py

```diff
@@ -0,0 +1,122 @@
+#!/usr/bin/python
+
+#    Copyright 2023 Red Hat, Inc.
+#
+#    Licensed under the Apache License, Version 2.0 (the "License"); you may
+#    not use this file except in compliance with the License. You may obtain
+#    a copy of the License at
+#
+#         http://www.apache.org/licenses/LICENSE-2.0
+#
+#    Unless required by applicable law or agreed to in writing, software
+#    distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+#    WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+#    License for the specific language governing permissions and limitations
+#    under the License.
+
+import csv
+import json
+import logging
+import pathlib
+from dataclasses import asdict, fields
+from typing import Dict, List, Optional
+
+import trestle.tasks.csv_to_oscal_cd as csv_to_oscal_cd
+import yaml
```

Comment:
For continuity, can you use this YAML library? Sticking the ones used by trestle - https://pypi.org/project/ruamel.yaml/.